README.md

```diff
@@ -28,31 +28,35 @@ To-do list is a complete, full-stack example of Stigg integration:
 
 You'll need a Stigg account in order to get Client/Server API key and run this project.
 
-## Getting started
-
-Install dependencies using yarn:
-```
-yarn
-```
-
-Add .env file in **client** folder with the following key:
-```
- NX_STIGG_CLIENT_API_KEY=<YOUR-CLIENT-API-KEY>
-```
-Add .env file in **server** folder with the following key:
-```
- NX_STIGG_SERVER_API_KEY=<YOUR-SERVER-API-KEY>
-```
-
-If this is your first time running the app, you'll need to set up the app:
-```
-yarn setup
-```
 #### Billing integration
 In order to test this example project with Stripe integration, you'll need to [connect your Stripe account through the Stigg management UI.](https://docs.stigg.io/docs/stripe#adding-the-stripe-integration-in-stigg)
 
+## Getting started
 
-Run the app (starts both the client and server):
+### Setup
+* Install dependencies using yarn:
+  ```
+  yarn
+  ```
+* Add `.env` file with the following content:
+  ```
+  DATABASE_URL="file:./dev.db"
+  ```
+* Add `.env` file under `apps/client/` folder with the following key:
+  ```
+  NX_STIGG_CLIENT_API_KEY=<YOUR-CLIENT-API-KEY>
+  ```
+* Add `.env` file under `apps/server/` folder with the following key:
+  ```
+  NX_STIGG_SERVER_API_KEY=<YOUR-SERVER-API-KEY>
+  ```
```

Comment:
cant the 3 of those files merged in a single `.env` file with placeholders? (you can commit it once and then .gitignore any changes to it)

Reply:
It's possible to have all environment variables inside single `.env` (instead under `apps/server` and `apps/client` folders).
I also think it would be much simpler.

@nadavkaner what do you think?

And I'm not sure if understood you correctly - anyway, I don't want the `.env` file to be tracked by git at all, as of best practice and to avoid contradiction in environment variable configured in Heroku.

Reply:
hmm what I see others do is to have a `.env.example` file that is committed with placeholders:
https://github.com/stripe/stripe-billing-typographic#getting-started
https://github.com/hasura/hasura-ecommerce#getting-started

Reply:
I saw this pattern of having a "placeholder" .env file with the keys but with empty values committed to git. After adding the file to .gitignore, the local changes to it weren't tracked so it wont be committed by accident.

Reply:
I read about this and saw https://stackoverflow.com/a/63944977
I prefer to go with the `.env.example` file